Sources/MongoSwift/MongoCollection+Write.swift

```diff
@@ -53,10 +54,18 @@ extension MongoCollection {
         let reply = Document()
         var error = bson_error_t()
         if !mongoc_collection_insert_many(
+             // TODO SWIFT-139: include writeErrors and writeConcernErrors from reply in the error
```

Comment:
I think this comment should be moved down a line (above `throw`), as you did for `insert_one`.

Reply:
yup, good catch.